AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/FocusCanvasView.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package com.microsoft.accessibilityinsightsforandroidservice;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.view.View;
+import java.util.ArrayList;
+
+public class FocusCanvasView extends View {
+  private ArrayList<FocusElementHighlight> focusElementHighlights;
+  private ArrayList<FocusElementLine> focusElementLines;
+
+  public FocusCanvasView(Context context) {
+    super(context);
+    this.focusElementHighlights = new ArrayList<>();
```

Comment:
Seems like these are set externally so we should not need to create them here?

Reply:
I guess if there is a possibility of the draw method being invoked when these are not set, then we should specifically handle that scenario there instead of this.

Reply:
You're right, I've now removed that and handled the null case on draw. Good catch!